quickstart/dotnet/README.md

```diff
@@ -19,22 +19,41 @@ This sample shows you how to incorporate Azure Cache for Redis into a C# .NET Fr
 - Azure Cache for Redis cache - [create one](https://docs.microsoft.com/azure/azure-cache-for-redis/quickstart-create-redis)
 - [Visual Studio 2019](https://www.visualstudio.com/downloads/)
 - [.NET Framework 4 or higher](https://www.microsoft.com/net/download/dotnet-framework-runtime), which is required by the StackExchange.Redis client.
+- [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
 
 ## Set up the working environment
 
+### 1. Set up local credential for using Entra ID
+This sample uses Microsoft Entra ID for connecting to an Azure Cache for Redis instance.
+The following line of code in *ContosoTeamStats/RedisConnection.cs* obtains the default credential from your local machine or an Azure resource as the identity for authentication and authorization.
```

Comment:
Nit: use backticks `ContosoTeamStats/RedisConnection.cs` instead of italics *ContosoTeamStats/RedisConnection.cs*

Reply:
fixed